### PR TITLE
jrnl: update to 2.4.2.

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/jrnl/template [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
 archs=noarch
 build_style=python3-module
-pycompile_module="jrnl"
 hostmakedepends="python3-setuptools"
-depends="python3-cryptography python3-dateutil python3-keyring
-python3-keyrings-alt python3-parsedatetime python3-passlib python3-pytz
-python3-setuptools python3-six python3-tzlocal python3-xdg python3-yaml"
+depends="python3-ansiwrap python3-asteval python3-colorama python3-cryptography
+python3-dateutil python3-keyring python3-parsedatetime python3-passlib
+python3-pytz python3-tzlocal python3-xdg python3-yaml"
```
Comment (sgn) [inline srcpkgs/jrnl/template:10]: This adds too much noise.

Comment (travankor) [reply srcpkgs/jrnl/template:10]: to `git-blame` ?

I still end up changing 2/3 lines instead of 3/3

**Comment by ericonr** [comment]
Checking updates isn't working correctly. Would you mind adding an `update` file to `jrnl`?

**Comment by travankor** [comment]
Thanks, fixed.

**Comment by abenson** [comment]
This was missing at least 3 dependencies `python3-ansiwrap` `python3-asteval` don't exist, and it seems `jrnl` also needs `textwrap3`

**Comment by sgn** [comment]
That's my bad. travankor included all required dependencies. I forced-pushed without required dependencies.
Sorry both.

**Pull request merged**